### DISCUSSION TITLE
QR Error - Linear Photo mode with HERO12 Black

### DISCUSSION THREAD

**Discussion by fangchangma** [gopro/labs#941] [General]
Hi. Similar problem to #378 - when I scan the following QR code with my HERO12 Black the lens doesn't change to linear. https://gopro.github.io/labs/control/set/?cmd=mPfL0
![image](https://github.com/gopro/labs/assets/17073991/4e54319a-3646-474f-a145-a46d77333478)

https://gopro.github.io/labs/control/set/?cmd=mPfL doesn't work either.
![image](https://github.com/gopro/labs/assets/17073991/fbb03d81-fce5-401d-a145-fe9bde873efb)

**Comment by dnewman-gpsw** [comment]
Wow! I guess no one uses photo Linear with Labs, That has been broken since HERO9.
